src/backend/api/Fusion.Resources.Domain/Queries/GetContractPersonnelRequest.cs

```diff
@@ -53,14 +60,20 @@ public async Task<QueryPersonnelRequest> Handle(GetContractPersonnelRequest requ
                     .FirstOrDefaultAsync(r => r.Id == request.RequestId);
 
                 var basePosition = await orgResolver.ResolveBasePositionAsync(dbRequest.Position.BasePositionId);
-               
+
                 var position = new QueryPositionRequest(dbRequest.Position)
                     .WithResolvedBasePosition(basePosition);
 
                 var workflow = await mediator.Send(new GetRequestWorkflow(request.RequestId));
 
                 var returnItem = new QueryPersonnelRequest(dbRequest, position, workflow);
 
+                if (request.Query?.ShoudExpand("comments") ?? false)
```

Comment:
I've tried to make the expands a part of the query request, in an attempt to make it available in pragmatically executed queries.. Maybe check if that could be done here as well?

Reply:
Sorry, not quite sure what you mean here. Will talk about this one directly.

Reply:
Convert as much of the odataqueryparam in the query object as possible. This makes the functionality available programmatically (typo above :p). I tried a pattern that defines a private enum (flag-enabled) and private Expand property on the command.

That way the handler can reach it, and checking for expand is more robust/clean as you don't have to do all the string stuff..

One can also when required create public fluent methods on the query to set the expand ( .ExpandComments() ) in case one needs it in a internal query...

Reply:
Ref `GetContractPersonnelRequests`

Reply:
Implemented in similar pattern as `GetContractPersonellRequests`